pkg/forwarder/forwarder.go

```diff
@@ -148,7 +148,12 @@ func New(cfg Config) (*Worker, error) {
 
 	// Create the `fromClient`.
 	fromTransport := metricsclient.DefaultTransport()
-	if len(cfg.FromCAFile) > 0 {
+	fromClient := &http.Client{Transport: fromTransport}
+	if cfg.Debug {
+		level.Info(logger).Log("msg", "fromClient transport: Debug ")
```

Comment:
If the transport is Debug, shouldn't Log level match that as well? I'd just move that.

Reply:
Right